util/gvalid/gvalid_validator_check_struct.go

```diff
@@ -132,6 +132,12 @@ func (v *Validator) doCheckStruct(ctx context.Context, object interface{}) Error
 			fieldName       = field.Name()                  // Attribute name.
 			name, rule, msg = ParseTagValue(field.TagValue) // The `name` is different from `attribute alias`, which is used for validation only.
 		)
+		// It uses json tag value as alias name if exists.
+		if len(name) == 0 && v.jsonTagAsAlias {
+			if jsonTag := field.Tag("json"); len(jsonTag) > 0 {
```

Comment:
- Please consider there's json tag value pattern like `json:"name,omitempty"`
- Please add associated unit testing case for your changes.